### ISSUE TITLE
Create an 'About' page

### ISSUE THREAD

**Issue by uruvelakassapa** [bkhpanigha/hh-suttas#141]
Was thinking we could move the foreword and some instructions on the functionality of the site there.

**Comment by interested-in-dhamma** [comment]
Good idea. The "Foreword" feels a bit out of place now.

**Comment by Reptilioo** [comment]
Yes this is a good idea, right now I'm not sure most people know about the offline mode for example, or even that we can toggle pali (or the the side-by-side option when pressing "S") and the like
